internal/util/version_test.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var originalVersion string
+
+func TestGetLatestRelease(t *testing.T) {
```

Comment:
these are using the github api. why dont we mock the github api so we can test these methods?